includes/Context.php

```diff
@@ -164,11 +164,11 @@ public function get_reference_site_url() {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param int|\WP_Post $post  Optional. Post ID or post object. Default is the global `$post`.
+	 * @param int|\WP_Post $post  Post ID or post object. Optional. Default is the global `$post`.
```

Comment:
Per WP core standards, the `Optional.` piece should be the beginning of optional parameter descriptions, so this should be changed back.

Reply:
```suggestion
	 * @param int|\WP_Post $post  Optional. Post ID or post object. Default is the global `$post`.
```